examples/src/main.rs

```diff
@@ -30,37 +30,43 @@ fn main() {
 
     // instantiate and prepare the example
     let example = match options.example {
+        ExampleType::FibSmall { sequence_length } => {
+            fibonacci::fib_small::get_example(&options, sequence_length)
+        }
         ExampleType::Fib { sequence_length } => {
-            fibonacci::fib2::get_example(options, sequence_length)
+            fibonacci::fib2::get_example(&options, sequence_length)
         }
         ExampleType::Fib8 { sequence_length } => {
-            fibonacci::fib8::get_example(options, sequence_length)
+            fibonacci::fib8::get_example(&options, sequence_length)
         }
         ExampleType::Mulfib { sequence_length } => {
-            fibonacci::mulfib2::get_example(options, sequence_length)
+            fibonacci::mulfib2::get_example(&options, sequence_length)
         }
         ExampleType::Mulfib8 { sequence_length } => {
-            fibonacci::mulfib8::get_example(options, sequence_length)
+            fibonacci::mulfib8::get_example(&options, sequence_length)
         }
-        ExampleType::Vdf { num_steps } => vdf::regular::get_example(options, num_steps),
-        ExampleType::VdfExempt { num_steps } => vdf::exempt::get_example(options, num_steps),
-        ExampleType::Rescue { chain_length } => rescue::get_example(options, chain_length),
+        ExampleType::Vdf { num_steps } => vdf::regular::get_example(&options, num_steps),
+        ExampleType::VdfExempt { num_steps } => vdf::exempt::get_example(&options, num_steps),
+        ExampleType::Rescue { chain_length } => rescue::get_example(&options, chain_length),
         #[cfg(feature = "std")]
-        ExampleType::RescueRaps { chain_length } => rescue_raps::get_example(options, chain_length),
+        ExampleType::RescueRaps { chain_length } => {
+            rescue_raps::get_example(&options, chain_length)
+        }
         #[cfg(feature = "std")]
-        ExampleType::Merkle { tree_depth } => merkle::get_example(options, tree_depth),
+        ExampleType::Merkle { tree_depth } => merkle::get_example(&options, tree_depth),
         #[cfg(feature = "std")]
         ExampleType::LamportA { num_signatures } => {
-            lamport::aggregate::get_example(options, num_signatures)
+            lamport::aggregate::get_example(&options, num_signatures)
         }
         #[cfg(feature = "std")]
         ExampleType::LamportT { num_signers } => {
-            lamport::threshold::get_example(options, num_signers)
+            lamport::threshold::get_example(&options, num_signers)
         }
     };
```

Comment:
Can we put `.expect()` here to avoid doing `unwrap()` on line 69?

Reply:
I've added `.expect()` to all the example types.